### DISCUSSION TITLE
DROP VANISHED

### DISCUSSION THREAD

**Discussion by mikeyhodl** [showtime-xyz/showtime-frontend#1714] [Q&A] [answered]
### My nft that I dropped 9hrs ago hasa vaniched after being fully claimed.
This is the txn [polygon](https://polygonscan.com/token/0x6f05ad23a827cd874a5640d7a9e749602d6af37d?a=0x98b2E5c160A775e3D9E3b48626568B017B51F75B)

**Comment by 0xkarmacoma** [comment, marked as answer]
hey @mikeyhodl, can you please send your issue to help@showtime.xyz?
